src/EnergyPlus/ThermalComfort.cc

```diff
@@ -259,11 +259,15 @@ namespace ThermalComfort {
 	Array1D< ThermalComfortDataType > ThermalComfortData;
 	Array1D< AngleFactorData > AngleFactorList; // Angle Factor List data for each Angle Factor List
 
+	bool ASH55Flag( false );
+	Real64 runningAverageASH( 0.0 );
+
 	// Functions
 	void
 	clear_state()
 	{
 		FirstTimeFlag = true;
+		ASH55Flag = false;
```

Comment:
@lgu1234 If you want, you could undo the changes to ASH55Flag and use the existing DataHeatBalance::AdaptiveComfortRequested_ASH55 instead.